content/scripts/build.js

```diff
@@ -1436,7 +1418,9 @@ class Builder {
       locale: metadata.locale,
       summary: metadata.summary,
       slug: metadata.slug,
-      file: folder,
+      // It's important that this is a full absolute path so that it
+      // will work more universally across builder, server, and watcher.
+      file: path.resolve(folder),
```

Comment:
Since you're making this `file` absolute, we should make `source` below absolute as well.